tests/test_other.py

```diff
@@ -7725,6 +7725,9 @@ def test_single_file(self):
                   expect_wasm = wasm_enabled
                   expect_wast = debug_enabled and wasm_enabled
 
+                  # currently, the emterpreter always fails with JS output
```

Comment:
please mention why ("since we do not preload the emterpreter file, which in non-html we would need to do manually")